tests/functional/aws-node-sdk/package.json

```diff
@@ -20,7 +21,7 @@
   },
   "scripts": {
     "test-aws": "AWS_ON_AIR=true mocha",
-    "test": "mocha",
+    "test": "mocha -t 50000",
```

Comment:
@YiuRULE why did you set this timeout ? Was the default not enough for your listing tests ? I'd understand that, but it needs to be made clear :)